packages/soci-snapshotter/soci-snapshotter.spec

```
@@ -0,0 +1,53 @@
+%global gorepo soci-snapshotter
+%global gover 0.5.0
+%global rpmver %{gover}
+
+Name: %{_cross_os}soci-snapshotter
+Version: %{gover}
+Release: 1%{?dist}
+Summary: A containerd snapshotter plugin which enables lazy loading for OCI images.
+License: Apache-2.0
+URL: https://github.com/awslabs/soci-snapshotter
+Source0: https://github.com/awslabs/soci-snapshotter/archive/refs/tags/v0.5.0.tar.gz
```

Reuse the macros and use the other URL, so that only the macro at the top needs to be updated in future spec edits:
```suggestion
Source0: https://github.com/awslabs/%{gorepo}/archive/%{gover}/%{gorepo}-%{gover}.tar.gz
```